portfolio/src/main/webapp/memes.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About me</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+      
+        <iframe width="1280" height="708" src="https://www.youtube.com/embed/d0MU8g57LhQ">
+        </iframe>
+        <iframe width="1280" height="776" src="https://www.youtube.com/embed/8MWWXd-6dLM">
+        </iframe> 
+        <iframe width="1280" height="720" src="https://www.youtube.com/embed/K8MFie0l7Ww">
+        </iframe>
```

Comment:
I would also reduce the indentation here a bit, to make it clearer that these elements are only one level deeper that the body tag above.